src/components/pages/Todos.vue

```diff
@@ -428,25 +420,42 @@ export default {
   methods: {
     ...mapActions([
       'clearSelectedTasks',
-      'loadTodos',
+      'loadAggregatedPersonDaysOff',
       'loadOpenProductions',
+      'loadTodos',
       'removeTodoSearch',
       'saveTodoSearch',
       'setDayOff',
+      'setTimeSpent',
       'setTodoListScrollPosition',
       'setTodosSearch',
-      'setTimeSpent',
       'unsetDayOff'
     ]),
 
-    isTabActive(tab) {
+    isActiveTab(tab) {
       return this.currentSection === tab
     },
 
+    async loadData(forced = false) {
+      await this.loadTodos({
+        forced,
+        date: this.selectedDate,
+        callback: () => {
```

Comment:
mixing await and callback is weird.

Reply:
I refactored the function to no longer have callbacks